config/settings.yml.example

```diff
@@ -2,8 +2,15 @@
 # Path to dynflow database, leave blank for in-memory non-persistent database
 :database:
 
-# URL of the smart proxy, used for reporting back
-:callback_url: 'https://localhost:8443'
+# URL of the foreman, used for reporting back
+:foreman_url: 'http://localhost:3000'
+
+:console_auth: false
+
+# SSL settings for client authentication against smart proxy.
```

Comment:
Right, I'will change that to `'SSL settings for client authentication against Foreman.'` and merge the PR